src/Lykke.Job.Bitcoin/Functions/UpdateBalanceFunctions.cs

```diff
@@ -73,22 +81,20 @@ private async Task ProcessBlock(int height)
 
             foreach (var tx in getTxOutputAddr.Result)
             {
-                // filter only lykke related transactions
                 if (tx.destinationAddresses.Any(txOutAddress => txOutAddress == _hotWalletAddressSettings.HotWalletAddress
                                                                        || observableAddresses.Contains(txOutAddress)))
                 {
-                    //to get tx input we need to load full tx data
+                    _log.Info("Detected lykke related transaction in block", context: new { height, tx.txHash });
```

Comment:
```suggestion
                    _log.Info("Detected lykke related transaction in block", context: new { Height = height, TransactionHash = tx.txHash, ToAddress = txOutAddress });
```

This is to fit existing field names in common services. this will help to search data in kibana